Packs/MicrosoftGraphDeviceManagement/Integrations/MicrosoftGraphDeviceManagement/MicrosoftGraphDeviceManagement.py

```diff
@@ -283,6 +285,20 @@ def get_managed_device_command(client: MsGraphClient, args: dict) -> None:
     return_outputs(human_readable, entry_context, raw_response)
 
 
+def get_managed_device_physical_memory_command(client: MsGraphClient, args: dict) -> None:
+    device_id: str = str(args.get('device_id'))
+    raw_response, device_id = client.get_managed_device_physical_memory(device_id)
+
+    device: dict = build_device_object(raw_response)
```

Comment:
Why do we need to build an entire device object if all we get is the physical memory?

Reply:
slack